src/translations/en.global.json

```diff
@@ -8,5 +8,19 @@
     "account.create.success": "Congrats! This name is available.",
     "account.create.error": "Username is taken. Try something else.",
     "account.available.success": "User found.",
-    "account.available.error": "User not found."
+    "account.available.error": "User not found.",
+
+    "actions": {
+       "CreateAccount": "New account created: ${receiverId}",
+       "DeleteAccount": "Account deleted: ${receiverId}",
+       "DeployContract": "Contract deployed: ${receiverId}",
+       "FunctionCall": "Called method in contract: ${methodName}",
+       "Transfer": "Transferred ${deposit}Ⓝ to ${receiverId}",
```

Comment:
```${deposit}``` is currently in attoNear, needs to be in NEAR (attoNear/(10^18)). @vgrichina Do we have a helper function in NEARlib for this?

Reply:
we have https://github.com/nearprotocol/near-wallet/blob/2f7e707b3f5c96ff6de8b690d427882b8a97da75/src/components/common/Balance.js